README.md

```diff
@@ -1 +1,1038 @@
+
+Finally, remember that the script files are not expected to be edited directly in the containers. Thus, the scripts files are not copied by `bin/config.sh export` from the containers to the staging area and more importantly, they are not copied to the master directory with `bin/config.sh save`. This means that if you are changing script files in your staging area, you will need to remember to copy the good ones back to the master directory manually in order for them to be versioned.
+
+## <a id="chapter-050"></a>An Example of Scripting in ForgeRock Components
+
+[Back to the Top](#top)
+
+To compare environments provided by ForgeRock components, we will create a script that will make an HTTP call to an online service and receive a response in the form of JSON. For this purpose, as an example, we will visit a dummy API employee record at `http://jsonplaceholder.typicode.com/users/1`. This endpoint returns JSON, which the script _could_ evaluate against other data provided by the script's environment.
+
+> If you use server-side scripts to access an API over encrypted connection, make sure Java, the script engine is running on, trusts the underlying SSL/TLS certificate.
+
+## <a id="chapter-060"></a>[ForgeRock Access Management](https://www.forgerock.com/platform/identity-management) (AM)
+
+[Back to the Top](#top)
+
+AM documentation can be found at [https://backstage.forgerock.com/docs/am](https://backstage.forgerock.com/docs/am).
+
+AM provides authentication and authorization services, and custom scripts can be used to augment the default functionality.
+
+### Managing Scripts in AM
+
+The [Managing Scripts](https://backstage.forgerock.com/docs/am/6.5/authorization-guide/#manage-scripts) chapter in AM's docs shows how the scripts can be managed via REST and command line interfaces. This may be the most efficient way to manage scripts in automatically maintained environments; for example, in production deployments.
+
+At the same time, AM console provides a visual and easy to use interface for managing scripts and applying them to authentication and authorization events. We will use the console to apply a script to an authentication procedure in AM.
+
+### Client-Side Scripts
+
+When used for authentication in the front channel, AM allows for creating client-side scripts that would be executed in the user agent.
+
+The use case for a client-side script is collecting information about the user agent's properties and its environment: [Geolocation](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/geolocation), IP, and whatever else that could be collected with a custom script running in a browser. Thus, the script needs to be written in JavaScript compatible with the browser.
+
+From the client side, the data collected with the script can be submitted to the server and become available to the server-side scripts involved in the same authentication procedure.
```

Comment:
become available to the server-side scripts

Wouldn't it be more accurate to say:

become available to the server-side module/node (which might also be implemented with server-side scripting)

Reply:
That is probably correct, but could that data be used without a server-side script, is it presented in a way other than the `clientScriptOutputData` variable?